README.md

```diff
@@ -12,85 +12,101 @@
 - [Sass vs SCSS](#sass-vs-scss)
 - [Frameworks](#frameworks)
 - [Libraries and Mixins](#libraries-and-mixins)
+  - [Animation](#animation)
+  - [Color](#color)
   - [Grid](#grid)
+  - [Media Queries](#mediaqueries)
```

Comment:
Can you please fix this one to be:

```
- [Media Queries](#media-queries)
```
instead?

The link won't work if there's no `-` in place of the space between words (Try clicking it on your repo https://github.com/flexbox/awesome-scss to see).

See more info here: https://gist.github.com/asabaylus/3071099